packages/hash/api/src/model/util.ts

```diff
@@ -68,17 +75,7 @@ export const nilUuid = "00000000-0000-0000-0000-000000000000" as const;
  */
 export const workspaceAccountId = nilUuid;
 
-const workspaceAccountShortname = getRequiredEnv("WORKSPACE_ACCOUNT_SHORTNAME");
-
-/**
- * @todo: revisit how this URI is defined and obtained as this is a temporary solution
- *   https://app.asana.com/0/1200211978612931/1202848989198299/f
- */
-export const workspaceTypesNamespaceUri = `https://example.com/@${workspaceAccountShortname}/types`;
-
-export const blockprotocolTypesNamespaceUri =
-  "https://blockprotocol.org/@blockprotocol/types";
-
+/** @todo avoid this look up, we can just do "entity-type" | ... etc. */
```

Comment:
We're passing in magic strings, and this seems to just avoid us passing an `"entity-type"` string rather than `"entityType"`... I don't really see value in these so I'm proposing we just swap the top type to `: "entity-type" | "property-type" | ..`

Reply:
I'm just saying we're passing strings, so there's no need to stick to camelCase. They aren't variables haha

Reply:
I see the change is applied. Reasonable to me as we're just using it for `generateSchemaUri` :+1: